index.html

```diff
@@ -37,8 +37,9 @@
 
 <p>A <strong>Code of Ethics and Professional Conduct</strong> is useful to
 define accepted and acceptable behaviors and to promote high standards of
-professional practice. The goal of this code is to ensure transparency in moderation of the organzition and to an environment where everyone can participate without fear of harassment. It also provides a benchmark for self evaluation and
-acts as a vehicle for better identity of the organization.</p>
+professional practice. The goals of this code are to ensure transparent moderation of the organzition,
```

Comment:
it'd be nice to tidy up the spelling here:

s/organzition/organization

Reply:
@chaals I can't edit this directly because the pull request is opened from your fork - please could you fix the spelling here?

Reply:
done